book/4-clustering.tex

```diff
@@ -92,6 +92,42 @@ \subsection{Homogeneity Completeness V Measure}
 \section{Silhouette Score}
 \subsection{Silhouette Score}
 
+The Silhouette Coefficient is a measure of how well samples are clustered with samples that are similar to themselves.
+
+The Silhouette Coefficient s for a given sample is given as 
+
+\begin{center}
+\tikz{
+\node[inner sep=2pt, font=\Large] (a) {
+{
+$\displaystyle
+s = \frac{{\color{cyan}b} - {\color{nmlpurple}a}}{\max({\color{cyan}b}, {\color{nmlpurple}a})}
+$
+}
+};
+}
+\end{center}
+
+\textbf{a} : The mean distance between a sample and all other points in the same cluster (mean intra-cluster distance)
```

Comment:
If you need to make reference of the formula variables in the paragraphs, make sure to wrap the term in $$ instead of \textb{a}. Like in the example below.

<img width="595" alt="Screenshot 2024-10-16 at 14 43 26" src="https://github.com/user-attachments/assets/80105ca4-4bef-41da-9820-6de01ef99868">

In both cases, **a** and **b**, you are introducing and describing the variables. Ideally, this is summarized in the equation with the arrows, making it easier to refer to the terms in the first paragraphs.